include/deal.II/fe/fe_poly_face.templates.h

```diff
@@ -61,12 +61,15 @@ UpdateFlags
 FE_PolyFace<PolynomialType, dim, spacedim>::requires_update_flags(
   const UpdateFlags flags) const
 {
-  UpdateFlags out = flags & update_values;
-  if ((flags & update_gradients) != 0u)
+  // FIXME
+  UpdateFlags out =
+    static_cast<UpdateFlags>(static_cast<unsigned int>(flags) &
+                             static_cast<unsigned int>(update_values));
```

Comment:
I think you probably want to undo that again.

Reply:
Yes, of course, I will revert this and the deletion of the `operator&` functions if/when we are happy with the rest.